app/src/androidTest/java/com/github/se/cyrcle/ui/review/AllReviewsScreenTest.kt

```diff
@@ -32,11 +32,11 @@ class AllReviewsScreenTest {
 
   @Before
   fun setUp() {
-    parkingViewModel.selectParking(TestInstancesParking.parking2)
     reviewViewModel.addReview(TestInstancesReview.review4)
     reviewViewModel.addReview(TestInstancesReview.review3)
     reviewViewModel.addReview(TestInstancesReview.review2)
     reviewViewModel.addReview(TestInstancesReview.review1)
+    parkingViewModel.selectParking(TestInstancesParking.parking2)
```

Comment:
Is this part of the changes that fixes the bugs? If so, do you have an idea which one and why?

Reply:
If you mean moving this call to after the .addReview, then it makes no difference. I was trying to understand why the tests didn't pass, but turns out it wasn't because of this, it was because in the test we set the selected Parking to testInstances1 while all the test instances review are linked to the testinstances2.

TLDR : I didn't have to change this.

Reply:
That's what I had guessed, wanted to ensure that our implementation's functioning didn't depend on the order we did things which would be quite fragile in my opinion